Source/Relativity.Transfer.SDK.Sample/Relativity.Transfer.SDK.Sample.csproj

```diff
@@ -7,8 +7,8 @@
 
 	<ItemGroup>
 		<PackageReference Include="ByteSize" Version="2.0.0" />
-		<PackageReference Include="Newtonsoft.Json" Version="10.0.2" />
-		<PackageReference Include="Relativity.Transfer.SDK" Version="1.6.0" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
This fixes this vulnerability.
https://github.com/relativitydev/relativity-transfer-sdk-samples/security/dependabot/1